lib/rspec/support/ruby_features.rb

```diff
@@ -1,3 +1,5 @@
+require 'rbconfig'
```

Comment:
Well, I noticed that rspec-core loads rbconfig:

https://github.com/rspec/rspec-core/blob/v3.2.0/lib/rspec/core.rb#L5

...but it's not used in rspec-core anywhere.  Originally it was used for the OS detection that got moved here but the require never got moved.  It should be here since it is used here.  I will be removing it from rspec-core soon.